eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -180,19 +167,6 @@ jobs:
           publishJUnitResults: false
           goals: 'install'
 
-      # We only copy artifacts into the staging in nightly builds, we don't bother with this for PR validation builds.
-      - pwsh: |
-          copy -r target/staging $(Build.ArtifactStagingDirectory)
-          copy eng/code-quality-reports/src/main/resources/index.html $(Build.ArtifactStagingDirectory)
-          copy output/dependencies.html (Join-Path $(Build.ArtifactStagingDirectory) "staging")
-        displayName: 'Copy reports to artifact staging'
-        condition: and(succeeded(), in(variables['Build.Reason'], 'Schedule', 'Manual'))
-
-      - publish: $(Build.ArtifactStagingDirectory)
-        condition: succeededOrFailed()
```

Comment:
Where does the dependency report handling happen now?

Reply:
Its happening in the ```java - aggregate-reports``` pipeline (internal).